model/countries/AU/AU-formatting-rules.yaml

```diff
@@ -0,0 +1,63 @@
+formatting-rules:
+  address:
+    - street-address-alternative-1
+    - separator: "\n"
+    - locality1
+    - separator: " "
+    - admin-area1
+    - separator: " "
+    - postal-code
+    - separator: "\n"
+    - country-name
+    - skip: country # redundunt with country-name
+    - skip: street-address # reduntunt with street-address-alternative-1
+
+  street-address-alternative-1:
+    - in-building-location
+    - separator: "  " # using 2 spaces
+    - building-location
+
+  in-building-location:
+    - unit
+    - separator: " "
+    - floor
```

Comment:
I wonder if in the case of Australia we should let `floor` capture both the floor type and floor number? According to [1] there are many ways to reference Floor (e.g Level, Basement, Ground Floor, etc). Same applies to `unit`.

[1] https://auspost.com.au/content/dam/auspost_corp/media/documents/australia-post-addressing-standards-1999.pdf

Reply:
I'm also thinking that indicating prefix `Level'` might not work, since instead of Level, we might have other concepts as well.

Reply:
I thought more about this....

1) I think that I have never seen a website that asks for the type of your floor. So this is not a field type that makes sense to expose to websites.

2) If a website has a field <input autocomplete="floor"> we enter "7", "B", ... "level 7". If a website has a field <input autocomplete="street-address", we should enter "level 7" or just "basement" - or even more complex "1º andar".

3) I think that it would be ok, if we formatted a street address with "Level 7" and the user changes it to "Floor 7" that's perfectly fine. We should just learn from that - but we can learn from it at the level of the `in-building-location` if we recognize Floor and Level as stop words. WDYT?

With this in mind, my suggestion would be to use

```
  building-location:
    - building
    - separator: " "
    - prefix: "Level "
    - street
```

Reply:
Decomposed the unit. PTAL!

Reply:
Do you mean that for floor it will be autofilled as "Level 7" for user and not just 7?
Also, let's note down that for most of the websites from rater's we separated line explicitly for floor is very rare, though the floor + apartment can be entered at the same time (in address-line2), but not sure how descriptively user's actually enter the data for address-line2.

> I thought more about this....
> 
> 1. I think that I have never seen a website that asks for the type of your floor. So this is not a field type that makes sense to expose to websites.
> 2. If a website has a field  we enter "7", "B", ... "level 7". If a website has a field <input autocomplete="street-address", we should enter "level 7" or just "basement" - or even more complex "1º andar".
> 3. I think that it would be ok, if we formatted a street address with "Level 7" and the user changes it to "Floor 7" that's perfectly fine. We should just learn from that - but we can learn from it at the level of the `in-building-location` if we recognize Floor and Level as stop words. WDYT?
> 
> With this in mind, my suggestion would be to use
> 
> ```
>   building-location:
>     - building
>     - separator: " "
>     - prefix: "Level "
>     - street
> ```

Reply:
Currently, your model suggests that an <input autocomplete="floor"> would have "Level 7" filled because you store `floor: Level 7`. I think that's not desirable.

I don't understand the sentence "Also, let's note down that for most of the websites from rater's we separated line explicitly for floor is very rare, though the floor + apartment can be entered at the same time (in address-line2), but not sure how descriptively user's actually enter the data for address-line2."

Reply:
Also, from your suggestion above:
  
```
building-location:
    - building
    - separator: " "
    - prefix: "Level "
    - street
```
 I was thinking maybe it is better to have a prefix inside in-building-location

```
in-building-location:
    - unit
    - separator: " "
    - prefix: "Level"
    - floor

```

Reply:
yes of course... this was careless copy&pasting.

Reply:
Adjusted.